README.md

```diff
@@ -48,17 +48,26 @@ with:
 ```
 
 
-#### Function's secrtes
+#### Function secrets
 
-When you implement your Cognite Function, you may need to have additional `secrets` to talk to 3rd party services like Slack.
-To achieve that one could create following dictionary:
+When you implement your Cognite Function, you may need to have additional `secrets`, for example if you want to to talk to 3rd party services like Slack.
+To achieve this, you could create the following dictionary:
 ```json
-{ "slack_credentials": "secret credential" }
+{"slack-token": "123-my-secret-api-key"}
 ``` 
-then go to to https://www.base64encode.org/ or SHELL to encode your dict into a string ()
+Use your terminal to encode your credentials into a string:
 ```shell script
-% echo '{ "slack_credentials": "secret credential" }' | base64 
+$ echo '{"slack-token": "123-my-secret-api-key"}' | base64 
 eyAic2xhY2tfY3JlZGVudGlhbHMiOiAic2VjcmV0IGNyZWRlbnRpYWwiIH0K
 ```
-
+...or use Python if you don't have `base64` available in your system:
+```sh
+$ echo '{"slack-token": "123-my-secret-api-key"}' | python -m base64
+eyJzbGFjay10b2tlbiI6ICIxMjMtbXktc2VjcmV0LWFwaS1rZXkifQo=
+```
+To decode and verify it, do:
+$ echo eyJzbGFjay10b2tlbiI6ICIxMjMtbXktc2VjcmV0LWFwaS1rZXkifQo= | python -m base64 -d
+{"slack-token": "123-my-secret-api-key"}
+```
 Take that string and store it into GitHub secret (COGNITE_FUNCTION_SECRETS from example above, f.ex) 
+Notes: _Keys must be lowercase characters, numbers or dashes (-) and at most 15 characters. You can supply at most 5 secrets_
```

Comment:
is it something from Functions? (should we validate that in our code then?)

Reply:
Unsure, the API returns a well-formulated error 🤔